ncard_app/templates/detail_views/add_event.html

```diff
@@ -3,8 +3,10 @@
 {% block title %}Add Event{% endblock %}
 
 {% block content %}
+    <a class="close-btn" href="{% url 'list-events' %}">
+        <button type="button" class="btn-close" aria-label="Close"></button>
+    </a>
```

Comment:
It's a bit confusing having a button inside a link, and having two separate classes called close-btn and btn-close. There should be a way to do this using a single `<a>` element, styled to look like a button, similar to the Export button.

Reply:
you are right, made it into a single <a> element